strapi/strapi
# [v4] fix(plugin generator): support valid package names for pluginId

### What does it do?

Updates the current replace regex to support both scoped package and old strapi plugin names.

### Why is it needed?

 Currently the generated plugins only support packages that are scoped under the strapi org. Most plugin developers are not creating any plugins under the strapi org (for obvious reasons) making it something every plugin developer needs to update.

### How to test it?

- Generate a plugin with a valid npm name
- Enable the plugin in config/plugins.js
- Start strapi
- Navigate to the plugin list section
- Observe that only the actual plugin name remains in the plugin list

Alternatively I have created some sample test at the following link: https://regexr.com/6dce9, unsure of how long it lasts.

### Related issue(s)/PR(s)

N/A
